docs/references/plugin-compatibility.md

```diff
@@ -0,0 +1,134 @@
+|hmac-auth                       |  :white_check_mark:              |  :white_check_mark:                        |
+|http-log                        |  :white_check_mark:              |  :white_check_mark:                        |
+|ip-restriction                  |  :white_check_mark:              |  :white_check_mark:                        |
+|jwt                             |  :white_check_mark:              |  :white_check_mark:                        |
+|key-auth                        |  :white_check_mark:              |  :white_check_mark:                        |
+|kubernetes-sidecar-injector     |  :white_check_mark:              |  :white_check_mark:                        |
+|oauth2                          |  :white_check_mark:              |  :x:                                       |
+|prometheus                      |  :white_check_mark:              |  :white_check_mark:                        |
+|rate-limiting                   |  :white_check_mark:              |  :white_check_mark:                        |
+|request-termination             |  :white_check_mark:              |  :white_check_mark:                        |
+|request-transformer             |  :white_check_mark:              |  :white_check_mark:                        |
+|response-ratelimiting           |  :white_check_mark:              |  :white_check_mark:                        |
+|response-transformer            |  :white_check_mark:              |  :white_check_mark:                        |
+|syslog                          |  :white_check_mark:              |  :white_check_mark:                        |
+|tcp-log                         |  :white_check_mark:              |  :white_check_mark:                        |
+|udp-log                         |  :white_check_mark:              |  :white_check_mark:                        |
+|zipkin                          |  :white_check_mark:              |  :white_check_mark:                        |
+|application-registration        |  :white_check_mark:              |  :x:<sup>1</sup>                           |
+|canary release                  |  :white_check_mark:              |  :white_check_mark:                        |
+|collector                       |  :white_check_mark:              |  :white_check_mark:                        |
+|degraphql                       |  :white_check_mark:              |  :white_check_mark:                        |
+|exit-transformer                |  :white_check_mark:              |  :x:                                       |
+|forward-proxy                   |  :white_check_mark:              |  :white_check_mark:                        |
+|graphql-proxy-cache-advanced    |  :white_check_mark:              |  :white_check_mark:                        |
+|graphql-rate-limiting-advanced  |  :white_check_mark:              |  :white_check_mark:                        |
```

Comment:
This is currently not compatible because it requires a custom DAO. This will be compatible once #630 is available for general use.

Reply:
Do we intend to hold merge until 0.9 and an image that includes it are released? I don't think we should document it as available until it truly is.

Reply:
No. We want to merge this right away.
We should document this as not available in DB-less for the time being.